---
title: 
layout: page
---

 **_De novo_ initiation of epigenetic silencing**
Transposable Elements (TEs) are DNA fragments that can mobilize and mutate the genome, causing hundreds of diseases and progression of certain types of cancer. To prevent new mutations, the cell ‘flags’ TE regions with chemical modifications such as methylation of DNA (in plants and mammals), and histone H3 lysine 9 residues (conserved from yeast to humans). These modifications mark TE regions for long-term ‘constitutive heterochromatin’ transcriptional silencing and are epigenetically passed to daughter cells after division to prevent reactivation.
<div class="row">
    <div class="col-md-12">
        <div class="col-xs-offset-1 col-md-10">
            <img src="/images/Rddm-01.png"/>
        </div>
    </div>
</div>
Plants have become a premier system to molecularly dissect heterochromatin formation, due to intense research on the RNA-directed DNA methylation (RdDM) pathway. In the model plant Arabidopsis, de novo RdDM of new TEs occurs in three distinct stages – initiation, establishment and maintenance. In the establishment stage of the canonical RdDM pathway, RNA Pol IV-transcribed precursors from heterochromatic TE loci are processed into 24 nucleotide (nt) small interfering RNAs (siRNAs) through the actions of RNA DEPENDENT RNA POLYMERASE 2 (RDR2) and DICER LIKE 3 (DCL3). These 24 nt siRNAs are loaded onto ARGONAUTEs (AGO4/AGO6) and transported into the nucleus, where they bind Pol V-transcribed scaffold RNAs. This interaction recruits DOMAINS REARRANGED METHYLTRANSFERASE 2 (DRM2), which then catalyzes de novo 5-cytosine methylation of the associated chromatin DNA. This cytosine methylation is maintained by the cell through the actions of other DNA methyltransferases. Finally, recognition of methylated DNA by histone methyltransferases lead to histone H3K9 methylation and constitutive heterochromatin formation. Several minor non-canonical pathways also contribute to RdDM. For instance, Pol II-transcribed TE mRNAs can be processed in a Pol IV-independent manner, resulting in 21–22 nt siRNAs. These 21–22 nt siRNAs are loaded onto AGO6 and target active TE loci, where they rely on the presence of Pol V scaffold RNAs to trigger de novo DNA methylation. 

Multiple studies have addressed how heterochromatin is epigenetically maintained, but little is known about how cells initially detect and silence newly inserted TEs without prior chromatin marks. This project aims to undertand the mechanism and molecular players underlying de novo initiation of TE silencing. 
